onnxruntime_extensions/pnp/_nlp.py

```diff
@@ -35,10 +33,25 @@ def bert_tokenize(ctx, input_names, output_names, container, operator_name=None,
     if 'strip_accents' in kwargs:
         attrs['strip_accents'] = kwargs['strip_accents']
 
-    return make_custom_op(ctx, 'BertTokenizer', input_names,
+    return make_custom_op(ctx, name, input_names,
                           output_names, container, operator_name=operator_name, **attrs)
 
 
+@schema(inputs=((_dt.STRING, []),),
+        outputs=((_dt.INT64, []), (_dt.INT64, []), (_dt.INT64, [])))
+def bert_tokenizer(ctx, input_names, output_names, container, operator_name=None, **kwargs):
+    return create_bert_tokenizer(ctx, 'BertTokenizer', input_names, output_names,
+                                 container, operator_name=operator_name, **kwargs)
+
+
+@schema(inputs=((_dt.STRING, []),),
+        outputs=((_dt.INT64, []), (_dt.INT64, []), (_dt.INT64, [])))
+def bert_tokenizer_v2(ctx, input_names, output_names, container, operator_name=None, **kwargs):
+    return create_bert_tokenizer(ctx, 'HfBertTokenizer',
```

Comment:
Can we unify the name, either it starts with hf, or ends with v2?

Reply:
Done!